.travis.yml

```diff
@@ -12,8 +12,8 @@ script:
     - ./build.sh vendor
     - ./build.sh utils
     - ./build.sh protobuf_verify
-    - ./build.sh binaries
     - ./build.sh test
+    - ./build.sh binaries
```

Comment:
Brrr I meant to send this in a separate PR. @drrt this is just something I noticed while watching the travis builds, LMK if it doesn't make sense and/or if I should back it out